site/runSiteFooterScripts.ts

```diff
@@ -66,6 +67,9 @@ export const runSiteFooterScripts = (
             runCookiePreferencesManager()
             void runDetailsOnDemand()
             break
+        case SiteFooterContext.explorerIndexPage:
+            hydrateExplorerIndex()
```

Comment:
Ahh, you need a bunch more here: Definitely `runSiteNavigation`, `runCookiePreferencesManager`, maybe also `runSiteTools`.